imports/plugins/core/orders/client/containers/orderDashboardContainer.js

```diff
@@ -500,11 +503,11 @@ class OrderDashboardContainer extends Component {
     // status of each order in regard to the other statuses
     // TODO: optimise this process to avoid having this similar repetitive block of code across 4 methods
     selectedOrders.forEach((order) => {
-      // TODO: remove these hard-coded zero indexes to enable multiple shipments in marketplace
-      const orderWorkflow = order.shipping[0].workflow;
+      const orderWorkflow = getShippingInfo(order).workflow;
```

Comment:
What if this `getShippingInfo(order)` returns null?

Reply:
The way `getShippingInfo(order)` is written, it'll either return the correct shipping object or an empty object.